File: ex-backend/src/lib/models/blog_post.ex
defmodule Src.BlogPost do
    use Ecto.Schema
    import Ecto.Changeset
    alias Src.BlogPost

    @primary_key {:id, :id, autogenerate: true}

    schema "blog_posts" do
        field :author, :string
        field :category, :integer
        field :link, :string
        field :text, :string
        field :title, :string
        field :created_at, :utc_datetime
        field :updated_at, :utc_datetime
    end

    @doc false
    def changeset(post, attrs) do
        post
        |> cast(attrs, [:author, :category, :text, :title, :link])
        |> validate_required([:author, :category, :text, :title, :link])
    end
end

File: cs-backend/TekkenHub/TekkenHub/Models/BlogPost.cs
using System;

namespace TekkenHub.Models
{
    public class BlogPost
    {
        public Int64 Id { get; set; } 
        public string Title { get; set; }
        public string Author { get; set; }
        public BlogCategory Category { get; set; }
        public string Text  { get; set; }
        public string Link { get; set; }
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
        public DateTime UpdatedAt { get; set; } = DateTime.UtcNow;
    }
}